judge/create_cgroups.in

```diff
@@ -24,16 +25,14 @@ for i in cpuset memory; do
     mkdir -p $CGROUPBASE/$i
     if [ ! -d $CGROUPBASE/$i/ ]; then
         if ! mount -t cgroup -o$i $i $CGROUPBASE/$i/; then
-            echo "Error: Can not mount $i cgroup. Probably cgroup support is missing from running kernel. Unable to continue."
-            print_cgroup_instruction
+            print_cgroup_instruction "Error: Can not mount $i cgroup. Probably cgroup support is missing from running kernel. Unable to continue."
```

Comment:
Adding this error message changes what `print_cgroup_instruction` does, so I'd suggest renaming it to something like `cgroup_error_and_usage`.

Reply:
Fair point! Will update and then "un-draft".

Reply:
https://github.com/DOMjudge/domjudge/pull/2146/commits/52e13ee83f667a172bb00138f767cb3b9b77f382